src/main/java/uk/gov/justice/digital/hmpps/cmd/api/security/UserSecurityUtils.kt

```diff
@@ -6,32 +6,29 @@ import org.springframework.security.core.userdetails.UserDetails
 import org.springframework.stereotype.Component
 
 @Component
+@SuppressWarnings("ConstantConditions")
 class UserSecurityUtils : AuthenticationFacade {
-    val authentication: Authentication
-        get() = SecurityContextHolder.getContext().authentication
+  val authentication: Authentication
+    get() = SecurityContextHolder.getContext().authentication
 
-    override fun getCurrentUsername(): String? {
-        val username: String?
-        val userPrincipal = userPrincipal
-        if (userPrincipal is String) {
-            username = userPrincipal
-        } else if (userPrincipal is UserDetails) {
-            username = userPrincipal.username
-        } else if (userPrincipal is Map<*, *>) {
-            username = (userPrincipal["username"] as String?)!!
-        } else {
-            username = null
-        }
-        return username
+  override fun getCurrentUsername(): String? {
+    val username: String?
+    val userPrincipal = userPrincipal
+    if (userPrincipal is String) {
+      username = userPrincipal
+    } else if (userPrincipal is UserDetails) {
+      username = userPrincipal.username
+    } else if (userPrincipal is Map<*, *>) {
+      username = (userPrincipal["username"] as String?)!!
+    } else {
+      username = null
     }
+    return username
+  }
 
-    private val userPrincipal: Any?
-        get() {
-            var userPrincipal: Any? = null
-            val auth = authentication
-            if (auth != null) {
```

Comment:
Linter was convinced that the 'authentation' could never be null

Reply:
I just fixed a risk profiler error in /health endpoint because the authToken was null. Maybe check that /health works ok here?